lib/Doctrine/Common/Proxy/ProxyGenerator.php

```diff
@@ -497,7 +520,7 @@ private function generateMagicSet(ClassMetadata $class)
      * @param string \$name
      * @param mixed  \$value
```

Comment:
In case of `__get()` we generate correct inherited parameter names. We should either do the same here or not do it for `__get()` but we should stay consistent.

I would personally generate only `@inheritdoc` when parent exists, for all magic methods.

Reply:
Fixed.